packages/api/src/command/webhook/webhook.ts

```diff
@@ -23,7 +24,12 @@ type WebhookPingPayload = {
   ping: string;
 };
 
-export type WebhookMetadataPayload = { messageId: string; when: string; type: string };
+export type WebhookMetadataPayload = {
+  messageId: string;
+  when: string;
```

Comment:
timestamp already here. no need to add

Reply:
yeah, I think we can use this and not have a dedicated timestamp header